### DISCUSSION TITLE
Disable autocmd

### DISCUSSION THREAD

**Discussion by esniper** [LazyVim/LazyVim#1010] [Q&A] [answered]
I want to disable the autocmd that enables going to the last location when opening a file.

Specifically, I want to disable this only for git commit message editor. If I can't do that, I'd want to disable it all together.

**Comment by folke** [comment]
add to your autocmds:

```lua
vim.api.nvim_create_augroup("lazyvim_last_loc", {clear = true})
```

**Comment by folke** [comment, marked as answer]
I just  pushed a change to exclude `gitcommit` from last loc. Makes little sense indeed
